Tribler/Test/test_my_channel.py

```diff
@@ -125,6 +129,88 @@ def do_page():
 
         self.startTest(do_page)
 
+    def test_metadata(self):
+        def take_screenshot_quit():
+            self.screenshot('Home')
+            self.quit()
+
+        def check_home():
+            self.CallConditional(20, lambda: len(self.guiUtility.frame.home.aw_panel.list.GetItems()) == 1,
+                                 take_screenshot_quit, "No thumbnail torrent in Home panel")
+
+        def on_channel_torrent_created(subject, events, object_id, data_list):
+            add_metadata = False
```

Comment:
I'm not sure about this, there's way too much logic here. I would modify it to read a rss-feed from disk.
Moreover, what happened to the ffmpeg thumbnail creation stuffs? That should still be implemented right, after a download changes to seeding it should attempt to create thumbnails if the torrent is part of the channel a user "owns".